clients/include/bytes.hpp

```diff
@@ -199,6 +199,13 @@ constexpr double gemm_gbyte_count(int m, int n, int k)
     return (sizeof(T) * (m * k + n * k + m * n)) / 1e9;
 }
 
+/* \brief byte counts of TRMM */
+template <typename T>
+constexpr double trmm_gbyte_count(int m, int n, int k)
+{
+    return (sizeof(T) * (m * n + tri_count(k))) / 1e9;
+}
+
```

Comment:
I think this will need to change to:
if(HIPBLAS_SIDE_LEFT == side)
    return (sizeof(T) * (m*n*2 + m*m/2))/1e9
else
   return (sizeof(T) * (m*n*2 + n*n/2))/1e9

This is because trmm loads m*n matrix  B, stores m*n matrix C, and loads triangle matrix A. If  HIPBLAS_SIDE_LEFT == side the triangle matrix is m*m, if HIPBLAS_SIDE_LEFT != side the triangle matrix is n*n. We can approximate and say half the values in the triangle matrix need to be loaded. This same approximation is made in the legacy blas timing code to calculate flops.

Reply:
I'm not including the `m * n` store to C right now, I'll add that. The numeber of elements in the triangular matrix is currently using our function `tri_count(k)` which is `(k * (k + 1)) / 2`. I find it odd that the flop calculation is different in the legacy blas, maybe I'm overlooking something. I guess it should be consistent between flops.hpp and bytes.hpp either way.